roles/kubevirt-resources/tasks/provision.yaml

```diff
@@ -0,0 +1,37 @@
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+        and admin_user is defined
+        and admin_password is defined
+
+- name: Create {{ namespace }} namespace
+  shell: kubectl create namespace {{ namespace }}
+  ignore_errors : true
+
+- name: Add Privileged Policy
+  command: "oc adm policy add-scc-to-user privileged -z {{ item }} -n {{ namespace }}"
+  with_items:
+    - kubevirt-privileged
+    - kubevirt-controller
+    - kubevirt-infra  # For KubeVirt v0.2.0
+  when: cluster=="openshift"
+
+- name: Add Hostmount-anyuid Policy
+  command: "oc adm policy add-scc-to-user hostmount-anyuid -z kubevirt-infra -n {{ namespace }}"
+  when: cluster=="openshift"
+
+- name: Cheking for kubevirt.yaml template in {{ kubevirt_template_dir }}
```

Comment:
You used the imperative mood in the previous tasks' names, and why a gerund here?  s/Checking/Check